.gitignore

```diff
@@ -1,3 +1,4 @@
 *.swp
 Cargo.lock
 target/
+build/
```

Comment:
nit: This is probably no longer necessary.

Reply:
yeah, got it